content-signup/README.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Send follow-up emails automatically
+description: Send users in an online community follow up content based on their
+interests.
+material_icon: autorenew
+create_time: 2019-07-15
+update_time: 2019-09-17
+---
+
+Contributed by Tech and Eco, follow me on [Twitter](https://twitter.com/TechandEco)
+
+By using a Google Form, let users select topics from a predefined list of
+content that you offer, and store the submissions in a Google Sheet. The
+content is then automatically sent to their email using a Google Doc as the
+email's template. The template inserts the user's name and the online content
+they selected which can be a video, whitepaper, blog, PDF summary, or other
+resources referenced via a URL.
+
+The Google Form can be accessed on your website, email signature,
+instant message, or wherever you share the form's link.
+
+![Topics are selected in a form, and upon submit an email is sent](https://github.com/gsuitedevs/solutions/raw/master/content-signup/assets/content-signup.gif)
```

Comment:
https://cdn.jsdelivr.net/gh/gsuitedevs/solutions@master/content-signup/content-signup.gif 

None of the other solutions have an assets directory so why don't you move it up to the top level and use the URL above (once it's merged, that's where your gif will be hosted on the CDN)

Reply:
I learned that I can also use `..` to refer to files one level up as a relative path.